sat/cli/bootprep/input/image.py

```diff
@@ -229,16 +227,11 @@ def ims_base(self):
     def k8s_api(self):
         """kubernetes.client.CoreV1Api: a kubernetes core API client"""
         try:
-            with warnings.catch_warnings():
-                warnings.filterwarnings('ignore', category=YAMLLoadWarning)
-                load_kube_config()
-        # Earlier versions: FileNotFoundError; later versions: ConfigException
-        except (FileNotFoundError, ConfigException) as err:
+            return load_kube_api()
```

Comment:
You could probably remove the TODO comment on line 225 above since the code duplication has essentially been minimized now.